### PR TITLE
Include axlUSDC in Kava Lend TVL

### PR DESCRIPTION
**Pull request by adamrturman**
## What changes

Adds axlUSDC to the TVL for Kava Lend (~$4.2 M at the time this PR was opened)

https://app.kava.io/lend

Screen Shot 2023-03-24 at 2 40 29 PM

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/hard.js exports TVL: 
        
 
 
    kava                      82.26 M
    kava-borrowed             8.05 M
    borrowed                  8.05 M
    
    total                    82.26 M

File: projects/hard.js [hunk 1]
```diff
@@ -18,7 +19,8 @@ const coingeckoIds = {
     [BUSD_DENOM]:'binance-usd',
     [XRPB_DENOM]:'ripple',
     [ATOM_DENOM]:'cosmos',
-    [AKT_DENOM]:'akash-network'
+    [AKT_DENOM]:'akash-network',
+    [AXLUSDC_DENOM]: 'axlusdc'
```
Comment (adamrturman) [inline projects/hard.js:23]: Is this the correct source for this id for the price? https://www.coingecko.com/en/coins/axelar-usdc

Comment (fahimahmedx) [reply projects/hard.js:23]: yes, that looks right.

**Comment by g1nt0ki** [comment]
@adamrturman thanks for the PR

**Pull request merged**